other-docs/guides/getting-help-with-altis.md

```diff
@@ -1,50 +1,91 @@
 
 ### Incident
 
-Incidents are unplanned events that negatively impact service, or quality of service of Altis. Be this part of the hosting infrastructure, or on the software side of Altis, if something isn’t getting the expected result, this would be an Incident, and requires investigation.
+Incidents are unplanned events that negatively impact service, or quality of service of Altis. Be this part of the hosting infrastructure, or on the software side of Altis, if there is an issue which is causing an immediate business impact, this would be an Incident, and you can make a support request to help investigate.
 
-From the support side, Incidents get a rapid response. If the site is down, this is an Incident, with a priority of ‘Urgent’. Alarm bells will ring, and we’ll put out the fires to get things running again by any means necessary.
+From the support team side, Incidents get a rapid response. Alarm bells will ring, and we’ll put out the fires to get things running again by any means necessary.
 
-Incidents focus on immediate attention, with an emphasis on immediate remediation. Remediating an Incident can take the form of temporary patches, which brings us to the next support type.
+If the site is down for example, select the 'Incident' type, with a priority of ‘Urgent’.
 
 ### Problem
 
-Problems are the cause of, or potential cause of Incidents. Incidents revolve around something unexpected. If a site is down, that’s an Incident, but the Problem is a known cause of an issue you wish to track, and coordinate help with. Problems typically take longer to resolve, as an appropriate solution is thought out and worked on.
+Problems are the cause of, or can be the potential cause of future Incidents. If you're encountering a problem which could cause potential Incidents, let us know!
 
 ### Task 
 
-There’s an array of tasks that might you need help with. Typically, this might mean provisioning of infrastructure to accommodate some need, it might be providing access to some tool or service used by the team, or it could be to run an operation that requires a cloud engineer. Tasks should be well described.
+There’s an array of tasks that might you need help with. Typically, this might mean provisioning of infrastructure to accommodate some need, it might be providing access to some tool or service used by the team, or it could be to run an operation that requires a cloud engineer. 
+
+Tasks should be well described with detailed requirements and time scales. We will not be able to service same-day requests.
 
 ### Question
 
-This type is fairly self explanatory. Have a question for us? Ask!
+Have a question for us? Ask!
 
-If you have a technical question or require clarification on a feature, just open a support question.
+If you have a technical question or require clarification on a feature, just open a support question, and our support team will get back to you.
```

Comment:
I don't think the last comma is necessary because the "and our support team will get back to you" is not part of the list of things that might be Questions.
```suggestion
If you have a technical question or require clarification on a feature, just open a support question and our support team will get back to you.
```